### DISCUSSION TITLE
How to add two supervisor on different lines in USTC Latex template

### DISCUSSION THREAD

**Discussion by MOmarFarooq** [ustctug/ustcthesis#335] [General]
I'd like to add two professors in the main page both in Chinese and English, as listed below.
Principal supervisor: Prof. XXX
Direct Supervisor: Prof. YYY

Please guide me, I will be very grateful to you

regards.

**Comment by zepinglee** [comment]
You may write both names of the professors separated by an English comma in the `supervisor` field.
https://github.com/ustctug/ustcthesis/blob/e498ea8e1ac8834d4da2290bb07c53b59bcdf33f/ustcsetup.tex#L11-L12
